include/mbedtls/config.h

```diff
@@ -137,12 +137,25 @@
 /**
  * \def MBEDTLS_HAVE_TIME_DATE
  *
- * System has time.h and time(), gmtime() and the clock is correct.
+ * System has time.h, time(), an implementation for mbedtls_platform_gmtime_r()
+ * (see below), and the clock is correct.
```

Comment:
Preexisting: “the clock is correct” in the first sentence is redundant with “The time needs to be correct” in the second sentence. I recommend removing “the clock is correct” from the first sentence.

Reply:
Addressed in 4e67cca.